browser/src/control/Toolbar.js

```diff
@@ -656,10 +656,46 @@ L.Map.include({
 		box.insertBefore(innerDiv, box.firstChild);
 		innerDiv.innerHTML = content.outerHTML;
 
-		var form = document.getElementById('modal-dialog-about-dialog-box');
+		var form = document.getElementById('about-dialog-box');
+
 		form.addEventListener('click', this.aboutDialogClickHandler.bind(this));
 		form.addEventListener('keyup', this.aboutDialogKeyHandler.bind(this));
 		form.querySelector('#coolwsd-version').querySelector('a').focus();
+		var copyversion = L.DomUtil.create('button', 'ui-pushbutton jsdialog', null);
+		copyversion.setAttribute('id', 'modal-dialog-about-dialog-box-copybutton');
+		copyversion.innerHTML = 'Copy version';
```

Comment:
'Copy version' should be _('Copy version') so it can be picked up by translations but other than that this looks great so, let's merge it + if the tests passed) and you can create a follow up PR for that string ;)